{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -5,6 +5,34 @@
   "title": "{{ cookiecutter.project_name}} v1.0.0",
   "type": "object",
   "properties": {
+    "_buckets": {
```

Comment:
it should be named `_bucket` to keep the consistency (also because it is only one bucket)

Reply:
👍